core/kt-osrd-utils/src/main/kotlin/fr/sncf/osrd/utils/indexing/StaticIdx.kt

```diff
@@ -36,6 +36,9 @@ value class OptStaticIdx<T>(private val data: UInt) {
             onSome(asIndex())
         }
     }
+
+    val box: StaticIdx<T>?
```

Comment:
nit: Maybe a clearer name (as box/unwrap is ambiguous in the context)
```suggestion
    val orNull: StaticIdx<T>?
```